src/components/Input/Input.protractor.driver.ts

```diff
@@ -1 +1 @@
-export * from 'wix-ui-core/dist/src/components/Input/Input.protractor.driver';
+export * from 'wix-ui-core/drivers/protractor';
```

Comment:
This looks incorrect. you should export only the Input's content

Reply:
fixed